WordPress-VIP-Go/ruleset.xml

```diff
@@ -7,7 +7,7 @@
 	<rule ref="WordPressVIPMinimum"/>
 
 	<!-- Things that may be incompatible with the VIP Go infrastructure and needs a dev to review -->
-	<rule ref="WordPressVIPMinimum.Functions.RestrictedFunctions.file_ops_delete">
+	<rule ref="WordPressVIPMinimum.Functions.RestrictedFunctions.file_ops_unlink">
```

Comment:
There's already an entry for unlink on line 80, so this entry can be deleted completely.

Reply:
Ow, good catch, thank you! And yes, I agree. While it doesn't do any harm (at this time) having the duplicate in the ruleset, it will make any debugging later more difficult, so let's just remove in both cases (ruleset + tests).

Reply:
Updated the PR to reflect this.